### PR TITLE
Add ruby sample apps

### PR DESCRIPTION
**Pull request by arjun024**
cc @thitch97 

~- When a builder is available at `gcr.io/paketo-buildpacks/ruby`, the builder must be replaced with it.~

### CODE REVIEW

File: ruby/passenger/README.md
Comment (kvedurmu) [inline ruby/passenger/README.md:5]:
```suggestion
`pack build passenger-sample --buildpack gcr.io/paketo-buildpacks/ruby`
```

File: ruby/puma/README.md
Comment (kvedurmu) [inline ruby/puma/README.md:5]:
```suggestion
`pack build puma-sample --buildpack gcr.io/paketo-buildpacks/ruby`
```

File: ruby/puma/README.md
Comment (kvedurmu) [inline ruby/puma/README.md:9]:
```suggestion
`docker run --interactive --tty --publish 9292:9292 puma-sample`
```

Note that this won't work because the buildpack doesn't currently support custom port binding (https://github.com/paketo-buildpacks/puma/issues/7). Probably better to just use port 9292 or alternatively modify rackup config.ru file to set the port to 8080

File: ruby/puma/README.md
Comment (kvedurmu) [inline ruby/puma/README.md:16]:
```suggestion
`curl http://localhost:9292`
```

see comment above

File: ruby/rackup/README.md
Comment (kvedurmu) [inline ruby/rackup/README.md:5]:
```suggestion
`pack build rackup-sample --buildpack gcr.io/paketo-buildpacks/ruby`
```

File: ruby/rake/README.md
Comment (kvedurmu) [inline ruby/rake/README.md:5]:
```suggestion
`pack build rake-sample --buildpack gcr.io/paketo-buildpacks/ruby`
```

File: ruby/thin/README.md
Comment (kvedurmu) [inline ruby/thin/README.md:5]:
```suggestion
`pack build thin-sample --buildpack gcr.io/paketo-buildpacks/ruby`
```

File: ruby/unicorn/README.md
Comment (kvedurmu) [inline ruby/unicorn/README.md:5]:
```suggestion
`pack build unicorn-sample --buildpack gcr.io/paketo-buildpacks/ruby`
```

**Review by kvedurmu — CHANGES_REQUESTED** [review]
Overall looks great. I think that we should just update the buildpack location to `gcr.io/paketo-buildpacks/ruby` and merge this once we have a new artifact published to this location.

Could y'all also update the [top level Readme](https://github.com/paketo-buildpacks/samples/tree/ruby) to link to these samples similar to what we do in the other languages?

**Review by kvedurmu — APPROVED** [review]

**Review by ForestEckhardt — APPROVED** [review]
Looks good to me would like @ryanmoran or @sophiewigmore to also approve before merging.

**Review by ryanmoran — APPROVED** [review]

**Pull request merged**
